vsphere/assets/configuration/spec.yaml

```diff
@@ -421,6 +425,10 @@ files:
         value:
           type: boolean
           example: true
+      - name: rest_api_options
+        description: REST API options
```

Comment:
Let's make it explicit that this is only for tags collection

Reply:
@FlorianVeaux Sure, good idea. Added a better desc here: https://github.com/DataDog/integrations-core/pull/9070/commits/7f48f2125efca9ac82b9f942104b78d2acbca04a